developer_manual/general/security.rst

```diff
@@ -1,51 +1,408 @@
 Security Guidelines
 ===================
 
-.. sectionauthor:: Bernhard Posselt <dev@bernhard-posselt.com>, Lukas Reschke <lukas@statuscode.ch>
+.. sectionauthor:: Bernhard Posselt <dev@bernhard-posselt.com>, Lukas Reschke <lukas@statuscode.ch>, Matthew Setter <matthew@matthewsetter.com>
+   
+Introduction
+------------
 
-This guideline highlights some of the most common security problems and how to prevent them. Please review your app if it contains any of the following security holes.
+These security guidelines are for core and application developers. 
+They give highlights some of the most common security problems and how to prevent them.
+They also give you some best practices and tips about security for ownCloud development.
+Please review your application, if it contains any of the following security holes.
```

Comment:
", and check if it..." ?
At the moment the sentence reads that you only need to review your application "if it contains any of the following security holes". Well, you won't know if it contains security holes unless you first review it :)

Reply:
Well said. Thanks for spotting that.